File: src/Data/ExternalEggSupply.java
package Data;

public class ExternalEggSupply extends Egg {
	private int numEggsSupplied = 0;

	public ExternalEggSupply(String numEggsSupplied) {
		this.numEggsSupplied = Integer.parseInt(numEggsSupplied);
	}

	ExternalEggSupply() {
		super();
	}

	private int getNumEggsSupplied() {
		return numEggsSupplied;
	}

	public String toString() {
		return "External Egg Supply \n" + getNumEggsSupplied();
	}

}

File: src/data/ExternalEggSupply.kt
package data

class ExternalEggSupply(numEggsSupplied: String) : Egg() {
    private var numEggsSupplied = 0

    init {
        this.numEggsSupplied = Integer.parseInt(numEggsSupplied)
    }

    override fun toString(): String {
        return "External Egg Supply \n$numEggsSupplied"
    }

}
